mavis/config.py

```diff
@@ -306,18 +306,18 @@ def __init__(self, **kwargs):
                 continue
             val = [v for v in re.split(r'[;\s]+', val) if v]
```

Comment:
if the val list is every empty this would crash.  Is that impossible?

Reply:
no, the parser requires 3-5 values for that option

Reply:
true. It should crash then anyway but I could make it a more helpful error message by checking and actually throwing an error